AVFoundation-CombineTests/TestSubscriber.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  TestSubscriber.swift
+//  AVFoundation-CombineTests
+//
+//  Created by József Vesza on 2020. 07. 24..
+//  Copyright © 2020. József Vesza. All rights reserved.
+//
+
+import Combine
+
```

Comment:
```suggestion
/// A `Susbcriber` Object that allows us to set its `demand` to arbitrary value and a closure when a completion has been emitted.
/// This class is used to unit test custom `Publisher` implementations.
```

Reply:
Added a slightly tweaked version of this